README.md

```diff
@@ -1,8 +1,173 @@
+
+## Fitting everything together
+
+![jaeger sequence](https://user-images.githubusercontent.com/2333426/49960015-1d716b80-fedd-11e8-98b8-48727f34414d.png)
+
+When we create an instance of the Jaeger client, we specify two time intervals, a queuing time interval and a reporting time interval. When the app starts a new span, the span stores the timestamp at the exact moment. Once the span is finished, the span is queued in memory for the duration of the queuing time interval. Once this timer is invalidates, the agent picks up the spans and persists that data to disk. Now the second reporting timer picks up the spans from disk, reports the spans through the sender and then flushes the cache.
+
+## Quick Start
+
+This section has quick setup steps for getting a simple proof of concept up and running. For a more comprehensive understanding of this project, Check out our [API reference](). A complete Opentracing solution consists of a client, an agent and a collector. The following steps will show you how to configure this project to report to a locally hosted Jaeger collector.
+
+###### Step 1 - Spin up a Jaeger collector
+If you already have a Jaeger collector up and running, you can skip this step. If not, you can run the Jaeger all-in-one docker image published to DockerHub by running the following command in Terminal. Once this step is completed, you should have a working Jaeger collector running locally.
+```
+$ docker run -d --name jaeger \
+-e COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
+-p 5775:5775/udp \
+-p 6831:6831/udp \
+-p 6832:6832/udp \
+-p 5778:5778 \
+-p 16686:16686 \
+-p 14268:14268 \
+-p 9411:9411 \
+jaegertracing/all-in-one:1.8
+```
+
+Now, if you open your browser and type [http://localhost:16686](http://localhost:16686), you should see the Jaeger web interface that displays trace information. Of course, there won't be any traces from our client there.
+
+###### Step 2 - Setup a mediator service
+The Jaeger collector only accepts data that is encoded in `Thrift` format. We have a mediator service that the client can report to. This mediator service accepts JSON spans from the Swift client, encodes this data to `Thrift Binary` and reports it to the Jaeger collector over UDP.
+
+* Download [our Jaeger mediator service](https://github.com/wayfair/jaeger-swift/tree/develop/Example/jaegerMediator).
+* By default, this mediator service reports spans to the locally hosted collector. If you already have a remote collector setup, you can specify the location in `configuration.json`.
+* Open Terminal. Run `npm ci` and then run `nom start`. You should now have the mediator service running on `http://localhost:3000`. You should see this message printed in the console
```

Comment:
message printed in the console ->message printed in the console.

Reply:
Updated